src/domain/Inputs/InputGroup/subcomponents/style.ts

```diff
@@ -36,6 +36,8 @@ const InputGroupButton = styled.button<IInputGroupButtonProps>`
         return css`border-radius: 4px 0 0 4px !important;`
       case 'right':
         return css`border-radius: 0 4px 4px 0 !important;`
+      case 'middle':
+        return css`border-radius: 4px 4px 4px 4px !important;`
```

Comment:
A 'middle' style in an input group would actually have no border radius 😅 

Also we really shouldn't be using the InputGroup.Button component at all outside of an InputGroup